### ISSUE TITLE
Improve screen reader announcement for daysHeader

### ISSUE THREAD

**Issue by fymmot** [mui-org/material-ui-pickers#2108]
- [x] I have searched the [issues](https://github.com/mui-org/material-ui-pickers/issues) of this repository and believe that this is not a duplicate.

## Summary 💡

To improve the user experience for screen reader users, I'm looking to hide the `daysHeader` section with `aria-hidden=true` but have not been able to find a way to do so.

![image](https://user-images.githubusercontent.com/6492106/92915300-b7f13c80-f42c-11ea-98f9-5be54c6a4da7.png)

Therefore I propose some way of accessing/customizing the daysHeader subcomponent, or possibly setting `aria-hidden=true` by default.

## Examples 🌈

N/A

## Motivation 🔦

The shortened weekday headers in `daysHeader` are presented as multiple text elements to screen readers, and are read as `"Mo" "Tu" "We" "Th" "Fr" "Sa" "Su"`.  

This does not provide much context if you cannot see the columns. Rather it creates a UX problem for screen reader users since it's tedious to navigate though seven non-meaningful text elements before getting to the calendar contents.  

For non-sighted users, I believe information about weekdays is best communicated in the `aria-label` for each dayButton. This can already be achieved with the renderDay prop, so no change is needed there.

**Comment by dmtrKovalenko** [comment]
I think it was fixed in v4. Check https://next.material-ui-pickers.dev/demo/datepicker

**Comment by fymmot** [comment]
@dmtrKovalenko 
Yes it is. Excellent! Thanks for the quick response!
